### ISSUE TITLE
Adding Quests?

### ISSUE THREAD

**Issue by piuy373** [xJon/The-1.12.2-Pack#93]
I really enjoyed how in the 1.7.10 pack had quests and had more of a concrete ending and guide me through the modpack.

**Comment by Riptide063** [comment]
I agree, a good questing mod always encourages me to use every aspect of a mod pack. It's the feature that I miss the most from the 1.7.10 pack.

**Comment by taz1009** [comment]
I absolutely love quests, helps me out so much on what to make next and where to go from my current position in the game 🙂

**Comment by Anonymous-Griefer** [comment]
I four this!
I would really love to see this pack adopt a Quest line! Making use of a mod such as http://www.9minecraft.net/better-questing-mod/ for example would be a great start!

**Comment by alexdevteam** [comment]
Bump. This would be really nice. I had to add Better Questing manually to my server and I'm doing the questlines myself. Would be great to have quests as a starting guide and as a reward system.

**Comment by Secr3ts** [comment]
bump. What's the ETA if there is one ?

**Comment by xJon** [comment]
There's no ETA currently

**Comment by xJon** [comment]
Just to add to this, if someone were to create quests I'd be very open to adding them to the modpack!
